mapserver/docker-compose.yml

```diff
@@ -4,23 +4,53 @@ services:
     image: camptocamp/mapserver:7.4
     restart: unless-stopped
 
+  rabbitmq:
+    image: library/rabbitmq
+    hostname: rabbitmq
+    environment:
+      - RABBIT_PASSWORD=rabbit_test_password
+      - USER=rabbit_user
+      - RABBITMQ_NODENAME=rabbit
+
+  worker:
+    build: django
```

Comment:
Make a settings file to configure environment where it is not possible to have separate worker instance (django and celery stays in the same server).